### PR TITLE
DBC22-2396: missing info display for regional weather

### PR DESCRIPTION
**Pull request by ray-oxd**
[DBC22-2396](https://jira.th.gov.bc.ca/browse/DBC22-2396)

This only affects regional weather stations. Local weather stations will be handled at a later time, but it likely won't be needed as there are no local weather stations that don't show either air or ground temp.

### CODE REVIEW

File: src/frontend/src/Components/map/layers/regionalWeatherLayer.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ export function getRegionalWeatherLayer(weatherData, projectionCode, mapContext,
               updateReferenceFeature(olFeatureForMap);
             }
           }
+
+          console.log('Weather data:', weather.forecast_group);
```
Comment (minORC) [inline src/frontend/src/Components/map/layers/regionalWeatherLayer.js:53]: Does this console log need to stay for any reason?

File: src/frontend/src/Components/map/panels/weather/RegionalWeatherPanel.js [hunk 2]
```diff
@@ -45,20 +53,30 @@ export default function RegionalWeatherPanel(props) {
...
+  const [expanded, setExpanded] = useState(false);
+
+  // Effects
   useEffect(() => {
     setSearchParams(new URLSearchParams({ type: 'regionalWeather', id: weather.id }));
   }, [feature]);
 
-  /* Main component */
+  useEffect(() => {
+    console.log('expanded', expanded);
```
Comment (minORC) [inline src/frontend/src/Components/map/panels/weather/RegionalWeatherPanel.js:68]: Same for this one, is it intentional?

**Review by minORC — CHANGES_REQUESTED** [review]
Looks good from doing blackbox testing. Just need to remove the unnecessary console logs.

File: src/frontend/src/Components/map/layers/regionalWeatherLayer.js (hunk 1 above)
Comment (ray-oxd) [reply src/frontend/src/Components/map/layers/regionalWeatherLayer.js]: removed in latest commit

File: src/frontend/src/Components/map/panels/weather/RegionalWeatherPanel.js (hunk 2 above)
Comment (ray-oxd) [reply src/frontend/src/Components/map/panels/weather/RegionalWeatherPanel.js]: removed in latest commit

**Review by minORC — APPROVED** [review]

**Pull request merged**